src/components/SUE/report/SueReportSend.tsx

```diff
@@ -158,6 +178,10 @@ export const SueReportSend = ({
                     }
                   />
                 </View>
+
+                {device.platform === 'android' && (
+                  <View style={{ height: normalize(keyboardHeight) * 0.8 }} />
```

Comment:
why 0.8?

Reply:
I found 0.8 by experimenting. It gives the right height for the keyboard.